src/util.js

```diff
@@ -1,4 +1,8 @@
 export function contains(arrayLike, stringOrRegex) {
+	if (!stringOrRegex) {
```

Comment:
This will return `false` if `stringOrRegex` is `''`, `null` and `undefined`. As long as that's what you were expecting!

Reply:
And if so, might be good to add test cases for those too

Reply:
Yeah, twas indeed the intent - but good call about adding tests though, added now! (on that note, dang there's a lot of copy-paste in these tests... definitely could do with some refactoring, but not doing that here 😄)